openownership/data-standard
# Update to readthedocs config file

# Overview

- What does this pull request do?

Updates the ReadTheDocs config file in line with the platform's [new requirements](https://blog.readthedocs.com/migrate-configuration-v2/).

- How can a reviewer test or examine your changes?

Eyeball the config file. (I've kept the python version at 3.9, not taken it to 3.11 as is in the RTD template. Is that ok?)

Verify that the build of this kd-od-rtd-update branch can be built on RTD (and still builds locally).

- Who is best placed to review it?

@rhiaro or @radix0000 

(Closes/Relates to) issue: #507 

## Translations

- N/A New or edited strings appearing as a result of this PR will be picked up for translation
- N/A I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

      https://openownership.github.io/bods-dev-handbook/standard_releases.html      

If we ever want to be able to rebuild the docs for older release branches, then we need to backport this update. @rhiaro or @radix0000 will need to advise on this.


- N/A I've updated the changelog, if necessary.
- N/A I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- N/A I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md
